astroquery/cadc/core.py

```diff
@@ -275,6 +276,162 @@ def get_collections(self):
                     append(row['energy_emBand'])
         return collections
 
+    @class_or_instance
+    def get_images(self, coordinates, radius,
+                   collection=None,
+                   get_url_list=False,
+                   show_progress=False):
+        """
+        A coordinate-based query function that returns a list of
+        fits files with cutouts around the passed in coordinates.
+
+        Parameters
+        ----------
+        coordinates : str or `astropy.coordinates`.
+            Coordinates around which to query.
+        radius : str or `astropy.units.Quantity`
+            The radius of the cone search AND cutout area.
+        collection: str, optional
+            Name of the CADC collection to query.
+        get_url_list: bool, optional
+            If true, returns the list of data urls rather than
+            the downloaded FITS files. Default is `False`.
+        show_progress: bool, optional
+            Whether to display a progress bar if the file is downloaded
+            from a remote server.  Default is `False`.
+
+        Returns
+        -------
+        list : A list of `~astropy.io.fits.HDUList` objects (or a list
+            of str if returning urls).
+        """
+
+        filenames = self.get_images_async(coordinates, radius, collection,
+                                          get_url_list, show_progress)
+
+        if get_url_list:
+            return filenames
+
+        images = []
+
+        for fn in filenames:
+            try:
+                images.append(fn.get_fits())
+            except HTTPError as err:
+                # Catch HTTPError if user is unauthorized to access file
+                logger.debug(
+                    "{} - Problem retrieving the file: {}".
+                    format(str(err), str(err.url)))
+                pass
+
+        return images
+
+    def get_images_async(self, coordinates, radius, collection=None,
+                         get_url_list=False, show_progress=False):
+        """
+        A coordinate-based query function that returns a list of
+        context managers with cutouts around the passed in coordinates.
+
+        Parameters
+        ----------
+        coordinates : str or `astropy.coordinates`.
+            Coordinates around which to query.
+        radius : str or `astropy.units.Quantity`
+            The radius of the cone search AND cutout area.
+        collection: str, optional
+            Name of the CADC collection to query.
+        get_url_list: bool, optional
+            If true, returns the list of data urls rather than
+            the list of context managers. Default is `False`.
+        show_progress: bool, optional
+            Whether to display a progress bar if the file is downloaded
+            from a remote server.  Default is `False`.
```

Comment:
might need to be ``False`` (double backticks) ?

Reply:
Ah, yes. That looks better!